package.json

```diff
@@ -1,114 +1,88 @@
 {
   "private": true,
   "name": "slate-packages",
+  "type": "module",
   "workspaces": [
     "packages/*"
   ],
+  "scripts": {
+    "build": "yarn build:rollup && yarn build:next",
+    "build:next": "cd ./site && next build",
+    "build:production": "cross-env NODE_ENV=production yarn build",
+    "build:rollup": "rollup --config ./config/rollup/rollup.config.js",
+    "clean": "rm -rf ./node_modules ./packages/*/{dist,lib,node_modules} ./site/.next",
+    "fix": "yarn fix:prettier",
+    "fix:eslint": "yarn lint:eslint --fix",
+    "fix:prettier": "yarn lint:prettier --write",
+    "lint": "yarn lint:eslint && yarn lint:prettier",
+    "lint:eslint": "eslint '**/*.{js,jsx,ts,tsx}'",
+    "lint:prettier": "prettier --list-different '**/*.{css,md,js,jsx,json,ts,tsx}'",
+    "open": "open http://localhost:3000",
+    "release": "yarn build:production && yarn test && yarn lint && lerna publish",
+    "serve": "cd ./site && next",
+    "start": "npm-run-all --parallel --print-label watch server",
+    "test": "cross-env BABEL_ENV=test FORBID_WARNINGS=true mocha --require ./config/babel/register ./packages/*/test/index.js",
+    "test:debug": "cross-env BABEL_ENV=test FORBID_WARNINGS=true mocha debug --require ./config/babel/register ./packages/*/test/index.js",
+    "watch": "yarn build --watch"
```

Comment:
I don't think `--watch` is a valid argument to `build` anymore now that the next build happens there.